### PR TITLE
add xkcd-cli

### PR DESCRIPTION
**Pull request by ocefpaf**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by ocefpaf** [comment]
I need this for... reasons.

**Comment by ocefpaf** [comment]
@conda-forge/staged-recipes, ready for review

File: recipes/xkcd-cli/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    - src
+  commands:
+    - pip check
+    - xkcd --help
+  requires:
+    - pip
+
+about:
+  license: MIT
+  license_file: LICENSE
```
Comment (jakirkham) [inline recipes/xkcd-cli/meta.yaml:40]: Looks like [this should be packaged in the future]( https://github.com/dotcs/xkcd-cli/commit/1d6f82fae4a9531c9d030b468315708a94af99ec#diff-50c86b7ed8ac2cf95bd48334961bf0530cdc77b5a56f852c5c61b89d735fd711R11-R13 ). Asked about a new release here ( https://github.com/dotcs/xkcd-cli/issues/6 )

Nothing to do here. Just noting for awareness

File: recipes/xkcd-cli/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
+{% set name = "xkcd-cli" %}
+{% set version = "0.0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/xkcd-cli-{{ version }}.tar.gz
```
Comment (jakirkham) [inline recipes/xkcd-cli/meta.yaml:9]: Do we want to inline `name` everywhere or use the Jinja variable?

No strong feelings either way. Just wanted to ask given we seem to be doing both

**Review by jakirkham — COMMENTED** [review]
Thanks Filipe! 🙏

Generally LGTM. Had one question below

**Comment by jakirkham** [comment]
Hey Filipe, saw you requested another review. Were there changes or comments on the questions above? If so, they seem not to be showing up for me. Just wanted to make sure I understand what we want to do here

**Comment by ocefpaf** [comment]
> Hey Filipe, saw you requested another review. Were there changes or comments on the questions above? If so, they seem not to be showing up for me. Just wanted to make sure I understand what we want to do here

Probably clicked there by mistake. No changes, I used grayskull here and I plan to keep using it to update the recipe afterwards, so changing this will only get clobbered later in the feedstock.

**Comment by jakirkham** [comment]
All good. Just want to make sure I'm not missing something

Ok let's merge then. Can raise an issue with Grayskull

**Comment by jakirkham** [comment]
Thanks Filipe! 🙏

**Pull request merged**